src/index.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable import/no-cycle */
+import './style.css';
+import showTasks from './showTasks.js';
+import { addToLocalStorage } from './setLocalStorage.js';
+import getTasksFromLocalStorage from './getTasks.js';
+import deleteTodo from './deleteOne.js';
+import removeCompletedTasks from './clearCompleted.js';
+
+const input = document.querySelector('.text');
+const form = document.getElementById('form');
+const removeCompleted = document.querySelector('.clear-completed');
+
+function clearInput() {
+  input.value = '';
+}
+
+// Add Task
+
+const addTodoTask = (e) => {
+  const tasks = getTasksFromLocalStorage();
+  e.preventDefault();
+
+  if (input.value === '') {
+    return;
+  }
+
+  const todo = {
+    description: input.value,
+    completed: false,
+    index: tasks.length + 1,
+  };
+
+  clearInput();
+  addToLocalStorage(todo);
+  showTasks();
+};
+
+const setIndex = (tasks) => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = i + 1;
+  }
+};
```

Comment:
I think it'll be good to use `forEach` to update the index here, I believe this will increase your code readability.

Reply:
Thank you for the review @ZeenatLawal 
I'll work on the requested changes now. 😃